venom_core/services/llm_simple_service.py

```
@@ -0,0 +1,1142 @@
+    """Bridge sync ONNX generator to async iteration without blocking event loop."""
+    queue: asyncio.Queue[str | BaseException | None] = asyncio.Queue()
+
+    def _producer() -> None:
+        try:
+            for token in client.stream_generate(
+                messages=messages,
+                max_new_tokens=max_new_tokens,
+                temperature=temperature,
+            ):
+                asyncio.run_coroutine_threadsafe(queue.put(token), loop)
+        except Exception as exc:  # pragma: no cover - safety bridge
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)